lib/rubocop/ast/node/send_node.rb

```diff
@@ -37,7 +37,8 @@ def lambda?
       #
       # @return [Boolean] whether this method is a staby lambda
       def stabby_lambda?
-        loc.selector.source == ARROW
+        selector = loc.selector
```

Comment:
Not sure why we need this local variable. It's even longer than just:

```ruby
loc.selector && loc.selector.source == ARROW
```

and doesn't seem to clear anything up, since the name is the same. 🙂